scene/main/node.h

```diff
@@ -455,6 +455,7 @@ class Node : public Object {
 	void add_child(Node *p_child, bool p_force_readable_name = false, InternalMode p_internal = INTERNAL_MODE_DISABLED);
 	void add_sibling(Node *p_sibling, bool p_force_readable_name = false);
 	void remove_child(Node *p_child);
+	void clear_children(bool p_deferred = true, bool p_include_internal = true);
```

Comment:
This should stay true. Internal children are supposed to be ignored only in scripts.